react-native-yoti-doc-scan.podspec

```diff
@@ -0,0 +1,23 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name           = 'react-native-yoti-doc-scan'
```

Comment:
@AdrienVioletYoti @eduardoyoti I can't tag Sam on here, do you know if this will also need to be changed to `@getyoti/react-native-yoti-doc-scan`?  I don't think it does, but might be worth just checking...

Reply:
It has been tested by QA this morning, after doing the so I guess it's fine but I will double check with Sam.